### ISSUE TITLE
Missing parts of ortho images when some views are disabled

### ISSUE THREAD

**Issue by CGSchwarzMayo** [pauldmccarthy/fsleyes#59]
I'm getting this on current git head (8eb1b3c13bda1b1962bbde093342c738b421ec0b) but not on an old release 0.31.2. Centos7, over x2go. 

The gif below is probably the best explanation, but in short, when using ortho layouts (any of them) without all 3 views enabled, parts of the image are missing. They can be brought back into view with extreme window ratios, but they disappear again when resized back. Forcing a redraw by changing slices does not fix it. The "rendering performance" setting does not affect it.

If you can't reproduce, I'm happy to test any proposed changes.

![fsleyes_layouts_bugs](https://user-images.githubusercontent.com/38332171/87813130-ec7aba80-c826-11ea-88cb-492155cbe608.gif)

**Comment by pauldmccarthy** [comment]
This looks like a refresh issueDid you install both versions in the same way? Or is the latest (broken) version installed via pip, and the older (working) version installed via conda?

**Comment by CGSchwarzMayo** [comment]
Paul,
You're absolutely right and I apologize for making apparently an invalid ticket. I checked out 0.31.2 in my git/dev environment and it still has the broken behavior, so this was not a regression.

My broken/development version was installed via:
```
pipenv shell
pip install --only-binary wxpython -f https://extras.wxpython.org/wxPython4/extras/linux/gtk2/centos-7/ wxpython
git clone https://github.com/pauldmccarthy/fsleyes.git fsleyes
cd fsleyes
pip install -e .
```
In that shell, wxpython was version 4.10.0

The older working version was installed via the standard fsl and fslpython install scripts. 
I dug around in that fslpython environment and found that the wxpython version was 4.0.6.

So I went back to my dev environment and did 
`pip install --only-binary wxpython -f https://extras.wxpython.org/wxPython4/extras/linux/gtk2/centos-7/ wxpython==4.0.6`
to downgrade that install to 4.0.6, and now it works as it should. 

In conclusion, this was a regression in wxpython centos7 distribution between version 4.0.6 and 4.10.0, and _not_ an fsleyes issue.

**Comment by pauldmccarthy** [comment]
Good to know, thanks for getting back to me. I think we will be sticking with wxpython 4.0.7 for the next FSL release, so hopefully this won't affect many people at the moment.
